__std__/cells/cloud/pipelines.nix

```diff
@@ -0,0 +1,50 @@
+{ cell
+, inputs
+}:
+let
+  inherit (inputs.tullia) flakeOutputTasks taskSequence;
+  inherit (inputs.nixpkgs.stdenv) system;
+  inherit (inputs.cells.automation) ciJobs;
+
+  common =
+    { config
+    , ...
+    }: {
+      preset = {
+        # needed on top-level task to set runtime options
+        nix.enable = true;
+
+        github-ci = {
+          # Tullia tasks can run locally or on Cicero.
+          # When no facts are present we know that we are running locally and vice versa.
+          # When running locally, the current directory is already bind-mounted
+          # into the container, so we don't need to fetch the source from GitHub
+          # and we don't want to report a GitHub status.
+          enable = config.actionRun.facts != { };
+          repo = "input-output-hk/plutus";
+          sha = config.preset.github-ci.lib.getRevision "GitHub event" null;
+        };
+      };
+    };
+
+  ciTasks = __mapAttrs
+    (_: flakeOutputTask: { ... }: {
+      imports = [ common flakeOutputTask ];
+
+      memory = 1024 * 8;
+      nomad.resources.cpu = 10000;
+    })
+    (flakeOutputTasks [ system "ciJobs" ] { outputs.${system}.ciJobs = ciJobs; });
+
+  ciTasksSeq = taskSequence "ci/" ciTasks (__attrNames ciTasks);
+in
+if system == "x86_64-linux" then
```

Comment:
Probably needs a big comment explaining that we don't support darwin yet but we will (?)

Reply:
What happens if we enable the darwin tasks? We'll just get a task that always fails? In fact, how does cicero know what kind of system to run a job on?

Reply:
I also wonder — this should not be needed. Cicero looks at `.#cicero.${builtins.currentSystem}`.
```suggestion
```

Reply:
No it is not needed, I just forgot to remove the line after some debugging

Reply:
@dermetfan nvm it looks like this is still needed, I'm not sure why but this invocation failed without it: https://cicero.ci.iog.io/invocation/a1be61dd-3452-4316-9e14-05748be735e4

Maybe I'm exporting the tasks wrong and darwin tasks are being lumped with linux tasks?

Reply:
I don't think this is a blocker, since cicero doesn't have darwin support yet

Reply:
I'd still like this to have a comment, even if it's "we need this and we're not sure why"

Reply:
oh sorry i completely missed your first message in this thread, ill add the comment in my next cicero PR